docs/src/main/asciidoc/_configprops.adoc

```diff
@@ -73,7 +73,7 @@
 |spring.cloud.kubernetes.discovery.service-labels |  | If set, then only the services matching these labels will be fetched from the Kubernetes API server.
 |spring.cloud.kubernetes.discovery.service-name | `unknown` | The service name of the local instance.
 |spring.cloud.kubernetes.discovery.wait-cache-ready | `true` | 
-|spring.cloud.kubernetes.enabled | `true` | Whether to enable Kubernetes integration.
+|spring.main.cloud-platform" | `KUBERNETES` | Whether to enable Kubernetes integration (this will force it and is not recommended)
```

Comment:
I dont think we should document this property here, its documented as part of Boot

Reply:
ok, I will remove this